OptionsWindow.xaml.cs

```diff
@@ -94,22 +96,35 @@ private void btn_BrowseForPath_Click(object sender, EventArgs e)
             }
         }
 
-        private void btn_Accept_Click(object sender, EventArgs e)
+        public void Refresh()
+        {
+            Array.ForEach(PathSettings(), setting => TextBoxForPathSetting(setting).Text = _options.Get(setting));
+
+            slider_DumpSpeedCD.Value = _options.preferredDumpSpeedCD;
+            slider_DumpSpeedDVD.Value = _options.preferredDumpSpeedDVD;
+        }
+
+        #region Event Handlers
+
+        private void OnAcceptClick(object sender, EventArgs e)
```

Comment:
Any particular reason you renamed this from `btn_Accept_Click`? It's not a rule, but in general, UI callbacks in C# are named after the UI elements that call them, since there can be multiple.

Reply:
Because my OCD is suffering, having an lowercase with underscore method name when everything is `CamelCased` really hurt me, can't we find a good trade of like `OnButtonAcceptClick`? `OnComboBoxDriveSpeedChange`? Just throwing ideas

Reply:
If we do this, I'd prefer if we standardized across all the callbacks. But that's my own OCD :P